src/argparser.py

```diff
@@ -392,31 +381,16 @@ def parse(my_parser):
         action="append",
         help="\t\tSpecify the raw data root dirs or desired results directory.",
     )
-    analyze_group.add_argument(
-        "-o",
-        "--output",
-        metavar="",
-        dest="output_file",
-        help="\t\tSpecify the output file.",
-    )
     analyze_group.add_argument(
         "--list-kernels",
         action="store_true",
-        help="\t\tList kernels. Top 10 kernels sorted by duration (descending order).",
+        help="\t\tList all detected kernels. Sorted by duration (descending order).",
     )
     analyze_group.add_argument(
         "--list-metrics",
         metavar="",
         choices=["gfx906", "gfx908", "gfx90a"],
-        help="\t\tList metrics can be customized to analyze on specific arch:\n\t\t   gfx906\n\t\t   gfx908\n\t\t   gfx90a",
-    )
-    analyze_group.add_argument(
-        "-b",
-        "--metric",
-        dest="filter_metrics",
-        metavar="",
-        nargs="+",
-        help="\t\tSpecify IP block/metric id(s) from --list-metrics for filtering.",
+        help="\t\tList all availible metrics for analysis on specified arch:\n\t\t   gfx906\n\t\t   gfx908\n\t\t   gfx90a",
```

Comment:
What do you think of this hard-coded list of arches?

Also, might as well fix the "available" typo.